Assets/UnZENity-Core/Scripts/UI/MenuManager.cs

```diff
@@ -0,0 +1,102 @@
+using System.Collections.Generic;
+using MyBox;
+using UnityEngine;
+
+namespace GUZ.Core.UnZENity_Core.Scripts.UI
+{
+    public class MenuManager : MonoBehaviour
```

Comment:
```suggestion
    public class MainMenu : MonoBehaviour
```

Please change the naming.
*Managers are always classes used from within GameGlobals. This MenuManager is also more like a MainMenuManager.

I see the other elements from within VRPlayerController. They are named *Menu only (without Manager suffix.)

```c#
        public MenuManager MainMenu;
        public QuestLogMenu QuestLogMenu;
        public StatusMenu StatusMenu;
```